### PR TITLE
added profile page #73

### LINKED ISSUE #73: Create profile page
**Is your feature request related to a problem? Please describe.**
Right now Libro doesn't have any profile page for the user.

**Describe the solution you'd like**
I would like to make a profile page if given the design.

### PR DESCRIPTION
**Pull request by amanxnanda**
### Related Issue
- Fixes #73

### Proposed Changes
- Change 1
- Added Profile page in the lib folder
- Change 2
- Added font_awesome_flutter package for icons

### Checklist
- [x] Tested
- [x] No Conflicts
- [x] Change In Code
- [ ] Change In Documentation

### CODE REVIEW

**Review by Souvikns — CHANGES_REQUESTED** [review]
Firstly, you have a color constant, re-allocate that to the `utils/constants.dart`, then import it from there.

**Comment by amanxnanda** [comment]
@Souvikns I thought so, for some reason I was getting an error while import. Anyway I'll try

**Review by infiniteoverflow — APPROVED** [review]

**Comment by infiniteoverflow** [comment]
@amanxnanda Work on the changes requested by Souvik , rest all looks good

File: pubspec.yaml [hunk 1]
```diff
@@ -23,12 +23,16 @@ environment:
...
-  flutter_facebook_login: null
+  flutter_facebook_login: ^3.0.0
   flutter_switch: ^0.1.6-beta.1
-  google_sign_in: null
+  font_awesome_flutter: ^9.0.0
+  google_fonts: ^2.0.0
+  google_sign_in: ^5.0.1
   hive: ^2.0.0
   hive_flutter: ^1.0.0
   google_fonts: ^2.0.0
```
Comment (infiniteoverflow) [inline pubspec.yaml:38]: google_fonts has been repeated in the pubspec.yaml file. Kindly remove this line.

**Review by infiniteoverflow — CHANGES_REQUESTED** [review]

**Comment by Souvikns** [comment]
@amanxnanda are you still working on this PR.

**Comment by infiniteoverflow** [comment]
@amanxnanda Are you still working on this PR ? There are some changes to make , can you look into it. If you are not able to , then we may have to assign someone else for the same.

**Pull request closed without merging**